components/surveys/responses/survey-responses-client.tsx

```
@@ -0,0 +1,207 @@
+				</Button>
+			</div>
+
+			{/* Filters */}
+			<div className="mb-6 bg-white dark:bg-gray-800 rounded-lg shadow-sm p-4">
+				<div className="flex items-center gap-4">
+					<div className="flex items-center gap-2">
+						<Filter className="w-4 h-4 text-gray-500" />
+						<span className="text-sm font-medium">Filters:</span>
+					</div>
+					<div className="flex items-center gap-2">
+						<label className="text-sm">From:</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)